### PR TITLE
fix extension/v1beta1 to apps/v1 for DaemonSet

### PR DESCRIPTION
**Pull request by yulng**
Fix extension/v1beta1 to apps/v1 for DaemonSet
According to the following document:
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/
image

Signed-off-by: yulng <wei.yang@daocloud.io>

### CODE REVIEW

File: node/windows-packaging/CalicoWindows/README.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
-Calico for Windows
```
Comment (caseydavenport) [inline node/windows-packaging/CalicoWindows/README.txt:1]: Could you revert the change to this file? It looks like it wasn't actually modified.

Comment (yulng) [reply node/windows-packaging/CalicoWindows/README.txt:1]: I try it

**Comment by yulng** [comment]
https://github.com/projectcalico/calico/pull/7287
